### ISSUE TITLE
wiki: email notifier

### ISSUE THREAD

**Issue by Dirk--** [OpenSeaMap/general#5]
Would like to receive an email notification on changes to watched properties
see https://www.mediawiki.org/wiki/Extension:Semantic_Watchlist
included at https://www.mediawiki.org/wiki/Semantic_Bundle
Thanks, Dirk

**Comment by ehrenfeu** [comment]
I'd also like to see this feature. Not receiving notifications upon updates is pretty annyoing.

**Comment by Dirk--** [comment]
Hi Niko,
thanks for your comment. I would label it 'feature request' rather than 'bug'.
Have a nice day, Dirk
http://maps.grade.de/

**Comment by ehrenfeu** [comment]
Hi @Dirk-- 
to me this is rather a mis- (or missing) configuration of the wiki. But feel free to adjust the labels ;)

Cheers
Niko

**Comment by Dirk--** [comment]
Niko, I do not have any idea how to assign or change labels @github :-(

**Comment by ehrenfeu** [comment]
Hm, I thought as the creator of the ticket, you were allowed to do so - but apparently you're not. Check if you can access the list of labels (click on the "Labels" text itself) shown in this image here:
![gh_labels](https://cloud.githubusercontent.com/assets/697157/5777086/c8fca0d6-9d8f-11e4-8e0f-d02ba2861282.jpg)

In the meantime, I'll add the label "feature" as well, as this is kind of a mixed situation :)

**Comment by Dirk--** [comment]
no 'gearwheel' on my side, think I need some kind of 'membership'...

**Comment by Skippern** [comment]
I think you need write-access to the repo to add new labels.

Aun Johnsen

**Comment by ehrenfeu** [comment]
@Skippern that's correct, I just thought the creator of a ticket could at least assign **existing** labels. But you can't.

**Comment by Skippern** [comment]
@ehrenfeu I think you are when creating the ticket, but maybe that is limited only upon creation, also that would be restricted to existing labels. I am not 100% sure though. I guess the train for adding labels have gone when you press the **Create Issue** button. But we are now weering totally off topic here.

**Mail Notification**
Somebody with access to the wiki-config should investigate the use of this extension and its impact on the wiki. I guess it would increase the usability and the interaction with the wiki from registered contributors.

**Comment by Dirk--** [comment]
ok, back to topic: 
Asked Matthias (admin of charlie.openseamap.org) on this: He seems to be 'not interested' :-(

**Comment by Skippern** [comment]
Not interested as in **don't think this is a good feature**, or as in **I don't have time to look at it now**?

**Comment by ehrenfeu** [comment]
@smurfix, maybe you could comment this?
